modules/core/test/v2/unit/coins/eos.ts

```diff
@@ -103,6 +106,67 @@ describe('EOS:', function() {
     });
   });
 
+  it('should create unsigned recovery transaction without Bitgo', async function() {
+    const userKey = 'xpub661MyMwAqRbcH1oUADxatLuKkVjaDB2zTNJoZQsGVQEvoogpbXJw24QMokNwFKj9Qhci6KWaCcQKrzpL4LCQXXX3YpTQxgD9KLBjhDrUWo4';
+    const backupKey = 'xpub661MyMwAqRbcH1n6sgY29G7dAxL7twS8rt1jyuuQb1kfnA7s3FJPGoVqb9JenXkeJmC4jZ8iVscn3AH6MkYAVc61FTYCHpxv5cxWar5Jw3C';
+    const rootAddress = 'i1skda3kso43';
+    const destinationAddress = 'ks13kdh245ls';
+
+    // load mock responses from json files
+    const rootResponse = JSON.parse(
+      fs.readFileSync(`${__dirname}/resources/eos/getAccountResponseSuccess1.json`).toString());
+    const desResponse = JSON.parse(
+      fs.readFileSync(`${__dirname}/resources/eos/getAccountResponseSuccess2.json`).toString());
+    const getInfoResponse = JSON.parse(
+      fs.readFileSync(`${__dirname}/resources/eos/getInfoResponseSuccess1.json`).toString());
+    const getBlockResponse = JSON.parse(
+      fs.readFileSync(`${__dirname}/resources/eos/getBlockResponseSuccess1.json`).toString());
+
+    // mock responses to the block chain
+    const callBack = sinon.createSandbox().stub(Eos.prototype, <any>'getDataFromNode');
+    callBack.withArgs({
+      endpoint: '/v1/chain/get_account',
+      payload: { account_name: rootAddress },
+    }).resolves(rootResponse);
+    callBack.withArgs({
+      endpoint: '/v1/chain/get_account',
+      payload: { account_name: destinationAddress },
+    }).resolves(desResponse);
+    callBack.withArgs({
+      endpoint: '/v1/chain/get_info',
+    }).resolves(getInfoResponse);
+    callBack.withArgs({
+      endpoint: '/v1/chain/get_block',
+      payload: { block_num_or_id: 191839472 },
+    }).resolves(getBlockResponse);
+
+    // can create unsigned recovery transaction
+    const unsignedRecoveryTransaction = await basecoin.recover({
+      userKey,
+      backupKey,
+      bitgoKey: 'key',
+      recoveryDestination: destinationAddress,
+      rootAddress,
+    });
+    unsignedRecoveryTransaction.txid.should
```

Comment:
Why assert the txid? Is that an important part of the test? It's better form to just assert on the properties that you want to hold true (destination and amount are the most important things to verify).

Reply:
txid is not really important, yea.